### DISCUSSION TITLE
Moderators should be able to convert comments to replies

### DISCUSSION THREAD

**Discussion by bennypowers** [community/community#38007] [Discussions]
1. A opens a discussion about `${FEATURE}`
2. discussion ensues, with many comments
3. B posts a comment suggesting to block `${FEATURE}` because of `${PROBLEM}`
4. C then posts a new comment asking questions about `${PROBLEM}`

---

Moderators should be able to move C's comment into B's comment replies, in other words, to convert C's comment to a reply.

This is analogous to how maintainers can convert issues to discussions, but without leaving the specific discussion thread.

**Comment by david-engelmann** [comment]
Sounds powerful, I like it
